cpp/src/ice2slice/Gen.cpp

```diff
@@ -649,7 +653,26 @@ Gen::TypesVisitor::visitEnum(const EnumPtr& p)
     }
     out.dec();
     out << nl << "}";
-    out << nl;
+}
+
+void
+Slice::Gen::TypesVisitor::visitConst(const ConstPtr& p)
+{
+    Output& out = getOutput(p);
+    out << sp;
+    out << nl << "// ice2slice could not convert:";
+    out << nl << "// const " << p->name() << " = " << p->value();
+
+    p->unit()->warning(p->file(), p->line(), WarningCategory::All, "ice2slice could not convert constant: " + p->name());
```

Comment:
Should we have a different category?

Reply:
I wouldn't be against adding another category, but I also don't think it's necessary here.